src/helpers/post-type-helper.php

```diff
@@ -33,4 +33,26 @@ public function is_indexable( $post_type_name ) {
 	public function get_public_post_types() {
 		return \get_post_types( [ 'public' => true ] );
 	}
+
+	/**
+	 * Removes all shortcode tags from the given content.
+	 *
+	 * @param string $content Content to remove all the shortcode tags from.
+	 *
+	 * @return string Content without shortcode tags.
+	 */
+	public function strip_shortcodes( $content ) {
+		return \strip_shortcodes( $content );
+	}
+
+	/**
+	 * Retrieves the post excerpt (without tags).
+	 *
+	 * @param int|\WP_Post $post Optional. Post ID or WP_Post object. Default is global $post.
```

Comment:
The argument isn't optional. You can leave `Option`. Also we aren't defaulting to the global `$post`, you can leave this as well.

Reply:
Right, I think I will add the `= null` part instead.